multi-agent/lib/mas/elements/tools/mcp_proxy/mcp_proxy_tool.py

```
@@ -74,43 +75,59 @@ def run(self, *args: Any, **kwargs: Any) -> Any:
             if not self._schema_initialized:
                 await self._ensure_tool_info(client)
 
-            # 4) Validate + prune arguments
             mcp_args = self._prepare_arguments(kwargs)
-            print(f"Executing tool '{self.mcp_tool_name}' with args: {mcp_args}")
 
-            # 5) Call tool using fresh client
             try:
                 result = await client.call_tool(self.mcp_tool_name, mcp_args)
-                print(f"Tool '{self.mcp_tool_name}' completed successfully")
             except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)